lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java

```diff
@@ -168,6 +177,18 @@ private void buildCompositionLayer() {
         this, Layer.Factory.newInstance(composition), composition.getLayers(), composition);
   }
 
+  private void applyColorFilters() {
+    if (compositionLayer == null) {
+      return;
+    }
+
+    ColorFilterData[] data = (ColorFilterData[]) colorFilterData.toArray();
```

Comment:
@gpeal colorFilterData is a Set. As far as I am aware there are only 2 ways to iterate the contents, iterator or create the array. I can change it to a List if you feel that is more appropriate. I don't think the list of color filters would get that huge anyway